jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -898,7 +901,7 @@ public void generateFiles(List<SubscriptionVH> subscriptions, String filePath) {
 		triggerNotificationforPermissionDenied();
 		//Below logger message is used by datadog to create notification in slack
 		if(Util.eCount.get()>0){
-			log.error("Error occurred in atleast one collector for jobId : Azure-Data-Collector-Job");
+			log.error("Error occurred in job azure-data-collector-job at least for one collector.");
```

Comment:
same here - it will be shown in logs of specific job

Reply:
can we say how many of them failed exactly?

Reply:
we are not interested in number of failures. Regardless of number of failures, we want to create only one data alert for one job run.
Please get an understanding on how datadog works and how data alerts are created.

Reply:
I don't care how DataDog works - we talking logs here, and data dog can handle them as we want it to handle them.
I'm asking here if we can add number of issues into log, so we know magnitude of the problem.

Reply:
"I don't know how DataDog works (nor that I care right now)" -- We technical people should have complete understanding on why each block of code has been written. Datadog working is not rocket science. Provide some strings and it will look for them in logs and create data alerts. We want to restrict to only one string so that we need not touch the settings in data dog again in future. Including the number of failures as you suggested.

"And if you want somebody to understand purpose of your changes - don't send them to read documentation and expect them to spend hours on that. Just explain bare minimum required for understanding." -- bare minimum is already present in the form of comment in line no 902. at least you should go through it.